From d2b89a4a6c73f08257f547cc43ccac8b91a7881a Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:53:44 -0400
Subject: [fs] ext4: fixup kerndoc annotation of mpage_map_and_submit_extent()

Message-id: <1414058125-4183-71-git-send-email-lczerner@redhat.com>
Patchwork-id: 98252
O-Subject: [RHEL 7.1 PATCH v2 070/171] ext4: fixup kerndoc annotation of mpage_map_and_submit_extent()
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Jan Kara <jack@suse.cz>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit 7534e854b930a021dedf9e16396ced5e70e1aba3

 Document give_up_on_write argument of mpage_map_and_submit_extent().

 Signed-off-by: Jan Kara <jack@suse.cz>
 Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/inode.c b/fs/ext4/inode.c
index ac4bfa3..d4817b2 100644
--- a/fs/ext4/inode.c
+++ b/fs/ext4/inode.c
@@ -2175,6 +2175,9 @@ static int mpage_map_one_extent(handle_t *handle, struct mpage_da_data *mpd)
  *
  * @handle - handle for journal operations
  * @mpd - extent to map
+ * @give_up_on_write - we set this to true iff there is a fatal error and there
+ *                     is no hope of writing the data. The caller should discard
+ *                     dirty pages to avoid infinite loops.
  *
  * The function maps extent starting at mpd->lblk of length mpd->len. If it is
  * delayed, blocks are allocated, if it is unwritten, we may need to convert
-- 
1.7.1